inner_code/filter_functions.py

```diff
@@ -0,0 +1,63 @@
+from typing import Tuple, Union
+def filter_gc_content(gc_bounds: Union [tuple, int], key: str, value: Union [tuple, int]):

```

Comment:
```suggestion
from typing import Tuple, Union


def filter_gc_content(gc_bounds: Union [tuple, int], key: str, value: Union [tuple, int]):

```

Ой, а почему типизация value - это кортеж или число?... в value у нас ведь записан кортеж из строк....